StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetFormFactorySnapshotTest.swift

```diff
@@ -410,6 +410,44 @@ final class PaymentSheetFormFactorySnapshotTest: STPSnapshotTestCase {
         XCTAssertTrue(formElement.validationState.isValid)
     }
 
+    func testEPM_subtitle() {
+        let configuration = PaymentSheet.Configuration()
+        let factory = factory(for: .card, configuration: configuration)
+        let formElement = factory.makeExternalPaymentMethodForm(subtitle: "Pay now with BufoPay", disableBillingDetailCollection: false)
+        let view = formElement.view
+        view.autosizeHeight(width: 375)
+        STPSnapshotVerifyView(view)
+        XCTAssertTrue(formElement.validationState.isValid)
+    }
+
+    func testEPM_subtitle_collectsBillingDetails() {
+        var configuration = PaymentSheet.Configuration()
+        configuration.billingDetailsCollectionConfiguration.name = .always
+        configuration.billingDetailsCollectionConfiguration.email = .always
+        configuration.billingDetailsCollectionConfiguration.phone = .always
+        configuration.billingDetailsCollectionConfiguration.address = .full
+        let factory = factory(for: .card, configuration: configuration)
+        let formElement = factory.makeExternalPaymentMethodForm(subtitle: "Pay now with BufoPay", disableBillingDetailCollection: false)
+        let view = formElement.view
+        view.autosizeHeight(width: 375)
+        STPSnapshotVerifyView(view)
+        XCTAssertFalse(formElement.validationState.isValid)
+    }
+
+    func testEPM_subtitle_doesNotCollectBillingDetails() {
```

Comment:
Maybe we should test :
1. `.address = auto` && disableBillingDetailCollection == true
2. `.address = auto` && disableBillingDetailCollection == false
3. `.address = never` && disableBillingDetailCollection == true
4. `.address = never` && disableBillingDetailCollection == false

Reply:
Good idea! Added!